docs/tutorials/personas.ipynb

```diff
@@ -0,0 +1,336 @@
+    "> inputs are set up in a way that all means- and health-tested transfers are assumed not\n",
+    "> to be taken up (while in reality, some of those transfers have very high take-up rates\n",
+    "> among low-income households).\n",
+    "\n",
+    "You can now compute taxes and transfers for the selected persona:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gettsim import InputData, MainTarget, TTTargets, main\n",
+    "\n",
+    "result = main(\n",
+    "    main_target=MainTarget.results.df_with_nested_columns,\n",
+    "    policy_date=example_persona.policy_date,\n",
+    "    input_data=InputData.tree(example_persona.input_data_tree),\n",
+    "    tt_targets=TTTargets.tree(example_persona.tt_targets_tree),\n",
+    "    include_warn_nodes=False,\n",
+    ")\n",
+    "\n",
+    "result"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4",
+   "metadata": {},
+   "source": [
+    "### Optional Arguments When Instantiating a Persona\n",
+    "\n",
+    "You can provide a grid of earnings levels to compute taxes and transfers across\n",
+    "different earnings levels. For example, we may want to look at net income in our example\n",
+    "household where the father earns 4,000 Euro per month. If working full-time, the mother\n",
+    "would earn the same amount. We want to investigate how total net income varies as she\n",
+    "changes her weekly hours between 25% and 75% of full-time work.\n",
+    "\n",
+    "To do this, create a `LinspaceGrid` object and specify the earnings range for each\n",
+    "`p_id`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gettsim_personas import einkommensteuer_sozialabgaben\n",
+    "\n",
+    "persona_with_varying_income_of_secondary_earner = einkommensteuer_sozialabgaben.Couple1Child(\n",
+    "    policy_date_str=\"2025-01-01\",\n",
+    "    bruttolohn_m_linspace_grid=einkommensteuer_sozialabgaben.Couple1Child.LinspaceGrid(\n",
+    "        p0=einkommensteuer_sozialabgaben.Couple1Child.LinspaceRange(\n",
+    "            bottom=4_000, top=4_000\n",
+    "        ),\n",
+    "        p1=einkommensteuer_sozialabgaben.Couple1Child.LinspaceRange(\n",
+    "            bottom=1_000, top=3_000\n",
+    "        ),\n",
+    "        p2=einkommensteuer_sozialabgaben.Couple1Child.LinspaceRange(bottom=0, top=0),\n",
+    "        n_points=100,\n",
+    "    ),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6",
+   "metadata": {},
+   "source": [
+    "Personas support different evaluation dates. Personas fix some input columns to constant\n",
+    "values (e.g. age at 30 years) and calculate the values of other input columns (e.g.\n",
+    "birth year as evaluation date minus 30 years). If no evaluation date is provided, the\n",
```

Comment:
```suggestion
    "birth year as evaluation date minus 30 years). In most cases, this will not matter much, but for many pension-related elements of taxes and transfers there often are birth year-based rules. If no evaluation date is provided, the\n",
```